draft-ietf-tls-esni.md

```diff
@@ -261,7 +261,10 @@ key_config
 : A `HpkeKeyConfig` structure carrying the configuration information associated
 with the HPKE public key. Note that this structure contains the `config_id`
 field, which applies to the entire ECHConfigContents. Sites MUST NOT publish
```

Comment:
"MUST NOT" -> "SHOULD NOT"? This sounds like it could be hard to enforce.

Reply:
Who are you thinking might enforce this? Clients? (I definitely don't feel strongly about the choice of word here.)